pkg/core/namespace.go

```diff
@@ -165,7 +171,12 @@ func (c *kubectlClient) NamespaceInit(manifests map[string]*Manifest, options Na
 		}
 
 		fmt.Printf("Applying %s in namespace %q\n", release, ns)
-		log, err := c.kubeCtl.Exec([]string{"apply", "-n", ns, "-f", resource})
+		resourceUrl, _ := url.Parse(resource)
+		labeledContent, err := c.kustomizer.ApplyLabel(resourceUrl, ownerLabel)
```

Comment:
Should we also apply this to every other resource that is applied from the manifest?

Reply:
I was focusing on namespace-related resources. Would you mind if I address this in another issue/PR?